app/views/reports/report.pdf.erb

```diff
@@ -9,7 +9,8 @@
   </head>
   <body class="print-report-body">
     <div class="print-report">
-      <%= sanitize_input(@html) %>
+      <% # Also whitelist <img> and <input type="checkbox"> tags %>
+      <%= sanitize_input(@html, ['img', 'input'], ['type', 'disabled', 'checked']) %>
```

Comment:
I think we don't allow `<img>` tags in rich text fields, on which we also use `sanitize_input` method. I guess that's why.